Editor/UI/Wizard/Setup/CreatorSetupWizard.cs

```diff
@@ -27,7 +31,7 @@ private static void ShowOnLoad()
             }
         }
 
-        [MenuItem("Innoactive/Run Training Setup Wizard...")]
+        [MenuItem("Innoactive/Creator/Create New Course...")]
```

Comment:
So, do we want to show the wizard every time a new course is created?

Reply:
Yes, that is intended and a UX decision.

Reply:
Well, don't believe me, but going through a setup wizard and especially setting the XR SDK every new training course feels like the totally going against scaling VR. 

[#JustMyTwoCents]() 😕

Reply:
The idea was to have it like this for now and then listen to users feedback. If it is annoying we can change it but this way you have to go through analytics as well. It is also not like you create a new course every 5 minutes.

Reply:
I am ok with the decision, I just wanted to point out my concern.
And yes, one might not create a new course every 5 minutes, but if someone uses the Creator is because that person is going to create several VR training apps.